src/test/resources/application.conf

```diff
@@ -0,0 +1,36 @@
+// a test DB is required.  Here's an example command to run one locally
```

Comment:
search in your email for an email from me titled "the perils of application.conf vs reference.conf" for more about this pain